<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_image">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读取图像</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="detect">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="ori_win">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>591</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vis_win">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>591</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>原图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结果图</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>基于YOLOv8的水下目标检测</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="load_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读取模型</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_text">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>load_image</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_image_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>detect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_model</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_model_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>690</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>721</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_image_clicked()</slot>
  <slot>detect_clicked()</slot>
  <slot>exit_clicked()</slot>
  <slot>load_model_clicked()</slot>
 </slots>
</ui>
